homeassistant/components/light/ads.py

```diff
@@ -0,0 +1,119 @@
+"""Support for ADS light sources."""
+
+import logging
+import math
+
+import voluptuous as vol
+
+from homeassistant.components.light import Light, ATTR_BRIGHTNESS, \
+    SUPPORT_BRIGHTNESS, PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.components.ads import DATA_ADS, CONF_ADSVAR
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+DEPENDENCIES = ['ads']
+DEFAULT_NAME = 'ADS Light'
+CONF_ADSVAR_BRIGHTNESS = 'adsvar_brightness'
+SUPPORT_ADS = SUPPORT_BRIGHTNESS
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ADSVAR): cv.string,
+    vol.Optional(CONF_ADSVAR_BRIGHTNESS): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the light platform for ADS."""
+    ads_hub = hass.data.get(DATA_ADS)
+    if not ads_hub:
+        return False
+
+    varname_enable = config.get(CONF_ADSVAR)
+    varname_brightness = config.get(CONF_ADSVAR_BRIGHTNESS)
+    devname = config.get(CONF_NAME)
```

Comment:
In the binary sensor platform you called this variable `name`. Please keep it consistent.

Reply:
Done.